plugins/techdocs/src/reader/components/TechDocsStateIndicator.tsx

```diff
@@ -21,7 +21,7 @@ import { Alert } from '@material-ui/lab';
 
 import { TechDocsBuildLogs } from './TechDocsBuildLogs';
 import { TechDocsNotFound } from './TechDocsNotFound';
-import { useTechDocsReader } from './TechDocsReaderPageContent';
+import { useTechDocsReader } from './TechDocsReaderPageContent/context';
```

Comment:
is this context very `TechDocsReaderPageContent` specific? As its imported in multiple components maybe worth moving out to `/components` ?